packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -257,14 +266,49 @@ const TransactionHeader = memo(
           width={20}
           onSelect={e => dispatchSelected({ type: 'select-all', event: e })}
         />
-        <Cell value="Date" width={110} />
-        {showAccount && <Cell value="Account" width="flex" />}
-        <Cell value="Payee" width="flex" />
+        <Cell
+          value="Date"
+          width={110}
+          icon={field === 'date' && ascDesc}
+          onClick={onSortTable('date')}
```

Comment:
This will call `onSortTable('date')` on every render, you’ll need to wrap it in a function in order for it to run on click. (same for all of the other ones)

```suggestion
          onClick={() => onSortTable('date')}
```

Reply:
🤦‍♂️ . Thanks.